qa/rpc-tests/p2p-compactblocks.py

```diff
@@ -164,6 +168,16 @@ def build_block_on_tip(self, node, segwit=False):
             block.rehash()
         if segwit:
             add_witness_commitment(block)
+        if 'rules' in blocktemplate and '!auxpow' in blocktemplate['rules']:
+            block.aux_pow = CAuxProofOfWork()
+            block.aux_pow.commit_version = block.nVersion
+            block.aux_pow.commit_hash_merkle_root = block.calc_commit_merkle_root()
+            block.aux_pow.commit_time = block.nTime
+            block.aux_pow.commit_bits = 0x207fffff # Will break after a difficulty adjustment...
```

Comment:
Maybe expand on comment to explain what should actually be done here to make this not break, even if only for your own future selves' sake.

Reply:
I changed the comment to "No difficulty adjustment on regtest." I think the comment was wrong before. It was warning that setting the auxiliary difficulty to a constant value wouldn't work once the difficulty adjusts. But the difficulty does not adjust in regtest mode.